features/frameworks/greentea-client/source/greentea_metrics.cpp

```diff
@@ -98,7 +98,10 @@ static void send_CPU_info()
     greentea_send_kv("__cpu_info        up time", stats.uptime);
     greentea_send_kv("__cpu_info     sleep time", stats.sleep_time);
     greentea_send_kv("__cpu_info deepsleep time", stats.deep_sleep_time);
-    greentea_send_kv("__cpu_info  %  sleep/deep", (stats.sleep_time * 100) / stats.uptime, (stats.deep_sleep_time * 100) / stats.uptime);
+    if(stats.uptime != 0) {
```

Comment:
`if (stats.uptime != 0) {` needs to  be, astyle has not checked as it ignores frameworks.

Reply:
if it's 0, then we are not sending kv, what do we do?

Reply:
Dunno. I think I've seen that case only during hal manager tests ... we're not supposed to be in uptime 0 as the system is actually running ...

Reply:
Understood. as uptime is 64bit, we do not expect having this 0, means here it would be ignored.

How was this discovered? Does greentea crash or ?

Reply:
I am looking at an alternative fix and maybe we can get rid of this check. Will test and update

Reply:
I made an update and this change is actually not needed.